setuputil.py

```diff
@@ -61,41 +61,53 @@
 ]
 
 server_requirements_table = {
-    'install_requires': clients_requirements_table['install_requires'] + [
-        'argcomplete',
+    'install_requires': [
+        'requests<=2.32.3',
+        'urllib3<=1.26.19',
+        'dogpile-cache<=1.2.2',
+        'packaging<=24.1',
+        'tabulate<=0.9.0',
+        'jsonschema<=4.23.0',
+        'dataclasses',
```

Comment:
Is there a reason we haven't set an upper limit for this (`dataclasses`)? If it's intentional, maybe adding a comment might be good.